src/core/components/InstallButton/InstallButton.scss

```diff
@@ -220,7 +220,7 @@ $installStripeColor2: #00c42e;
   &.enabled,
   &.installed {
     input + label::before {
-      background: $switchBackgroundOn url('../img/tick.svg') no-repeat 35% 50%;
+      background: $switchBackgroundOn url('./img/tick.svg') no-repeat 35% 50%;
```

Comment:
This doesn't appear to be used by any other component. If we need to use it later on, we could put it somewhere in a `core/img` folder or something.